app/editor/src/hooks/api-editor/useApi.ts

```diff
@@ -34,19 +34,19 @@ export const useApi = (
 
   return {
     request: (config: AxiosRequestConfig<any>) => {
-      return handleRequest(config.url, () => summon.request(config));
+      return handleRequest(config.url, () => summon?.request(config));
```

Comment:
Ran into this issue where it was `undefined` for some reason.  So I just added this to handle it.